_episodes/01-intro.md

```diff
@@ -139,7 +139,7 @@ weight in kilograms is now: 65.0
 > A variable is analogous to a sticky note with a name written on it:
 > assigning a value to a variable is like putting that sticky note on a particular value.
 >
-> ![Variables as Sticky Notes](../fig/python-sticky-note-variables-01.svg)
+> ![Variables as Sticky Notes analogy example depicting weight_kg value as 65.0](../fig/python-sticky-note-variables-01.svg)
```

Comment:
Per the webaim.org  suggestions, the `alt` attribute should be succinct, not add information not shown in the image, and does not need to repeat information given in the surrounding content. It also says it "is often a matter of personal interpretation" though, so it feels awkward to me to suggest changes to your interpretation, but based on the context of the image, do you think `value of 65.0 with weight_kg name stuck on it` (or some tweaked version of that) would be appropriate for more succinct text?

Reply:
Sure I will change, but I was aiming to illustrate also function of the image.

Reply:
Regarding [function discussed by WebAIM](https://webaim.org/techniques/alttext/#functional) I interpret to mean function such as if the image is also a link or clickable, what does clicking on it do. We could mention sticky notes explicitly, though, if that is what you were thinking as far as function: `Value of 65.0 with sticky note on it reading weight_kg`.

Reply:
I think, your suggestion is fine and we can leave it at that, unless someone has other comments or suggestions.